javascript/ql/src/Security/CWE-400/PrototypePollutionUtility.ql

```diff
@@ -23,8 +23,8 @@ import semmle.javascript.DynamicPropertyAccess
  */
 class SplitCall extends StringSplitCall {
   SplitCall() {
-    getSplitAt() = "." and
-    getUnsplit() instanceof ParameterNode
+    getSeparator() = "." and
+    getBaseString() instanceof ParameterNode
```

Comment:
this needs a `flowsTo`, doesn't it? If so, please add a test that would have caught it.

Reply:
An existing test did catch it, and I fixed it while you were writing that comment.